bcda/logging/middleware.go

```diff
@@ -136,3 +137,46 @@ func Redact(uri string) string {
 	}
 	return uri
 }
+
+type CommonLogCtxKeyType string
+
+const CommonLogCtxKey CommonLogCtxKeyType = "commonLog"
+
+func NewCommonLogFields(next http.Handler) http.Handler {
```

Comment:
nit — could we add some doc comments for these functions? I know the codebase in general doesn't really do this consistently, but I think it might help with maintainability since these are functions that will likely be used generally across the code.

Reply:
comments added!